hypertrace-core-graphql-context/build.gradle.kts

```diff
@@ -11,6 +11,7 @@ dependencies {
 
   implementation(project(":hypertrace-core-graphql-spi"))
   implementation("com.google.guava:guava")
+  implementation("org.hypertrace.core.grpcutils:grpc-context-utils")
```

Comment:
GRPC dependencies are not part of this module by design - it was meant to be independent of downstream impl details.